instructions/js/index.js

```diff
@@ -0,0 +1,17 @@
+const today = new Date();
+const thisYear = today.getFullYear();
+var footer = document.querySelector("#copyright");
+const copyright = document.createElement("p");
+copyright.innerHTML = ` roselynn ${thisYear}`;
+footer.appendChild(copyright);
+
+const skills = ['Html,Css and Javascript','Drupal','Q|A with webdriversIO','Service Now'];
+let skillsSection = document.getElementById("#skills")
```

Comment:
When you do `getElementById` you need to omit the leading `#`. `document.getElementById("skills")` and `document.querySelector("#skills") should both work here.